kong/clustering/wrpc_control_plane.lua

```diff
@@ -195,30 +208,17 @@ function _M:handle_cp_websocket()
     dp_version = dp_version,
     log_suffix = log_suffix,
     basic_info = nil,
-    basic_info_semaphore = semaphore.new()
+    basic_info_semaphore = semaphore.new(),
+    dp_plugins_map = empty_table,
```

Comment:
Is `dp_plugins_map` a constant table? or it will be changed later?

Reply:
Surely not. This is the initial value.

Reply:
As you said, then here we should not use `empty_table`, or else it will be referred by other objects.

I think it should be `{}`.

Reply:
> I think it should be `{}`.

+1

Reply:
As far as I know, this table is always read-only (and when its value changes, it's a whole new table). But it's OK to write `{}` just in case.